draft-ietf-quic-applicability.md

```diff
@@ -466,6 +466,9 @@ based on the Connection ID (see also {{connid}}) in the QUIC header,
 if present. This supports cases where address information changes, such as
 NAT rebinding, intentional change of the local interface, or based on an
 indication in the handshake of the server for a preferred address to be used.
+As such if the client knows or is likely to sit behind a NAT, use of the 
+Connection ID is strongly recommended. Further use of the Connection ID is
+generally recommendated when mitragtion is supported.
```

Comment:
```suggestion
generally recommendated when migration is supported.
```

Reply:
This is pretty woolly.  I think that you want to say why: identifying the connection that packets are destined for is more challenging if they don't include a connection ID and the source address might have changed.  An empty connection ID is only appropriate if the local address is unique, which opens up some negative consequences in terms of providing obvious linkability (though in practice this only works with limited numbers of connections on the target IP+port, so linkability is pretty much a given at that point.)

Reply:
I mainly wanted to give a recommendation that if you have no idea if you want/need a connection ID, you should probably have it (assuming that people agree that the right recommendation to give). I can add more explanation and also say something about linkability but I wanted to keep it simple... happy to take suggestions

Reply:
We're going to go ahead and merge this for the 109 document deadline submission; please open another issue / provide a PR / haunt us otherwise if this needs more discussion.